dev-python/rich-click/rich-click-1.7.2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+PYPI_NO_NORMALIZE=1
+inherit pypi distutils-r1
+
+DESCRIPTION="Format click help output nicely with rich"
+HOMEPAGE="https://pypi.org/project/rich-click/"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="dev-python/click[${PYTHON_USEDEP}]
+	dev-python/rich[${PYTHON_USEDEP}]
+	dev-python/typing-extensions[${PYTHON_USEDEP}]"
+
+distutils_enable_tests pytest
```

Comment:
We are either missing some test dependency here, or we may need to explicitly set some `PYTHONPATH`:
```
____________________________________ ERROR collecting tests/test_help.py _____________________________________
ImportError while importing test module '/tmp/portage/dev-python/rich-click-1.7.2/work/rich-click-1.7.2/tests/test_help.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.11/site-packages/_pytest/python.py:617: in _importtestmodule
mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
/usr/lib/python3.11/site-packages/_pytest/pathlib.py:567: in import_path
importlib.import_module(module_name)
/usr/lib/python3.11/importlib/__init__.py:126: in import_module
return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1204: in _gcd_import
???
<frozen importlib._bootstrap>:1176: in _find_and_load
???
<frozen importlib._bootstrap>:1147: in _find_and_load_unlocked
???
<frozen importlib._bootstrap>:690: in _load_unlocked
???
/usr/lib/python3.11/site-packages/_pytest/assertion/rewrite.py:186: in exec_module
exec(co, module.__dict__)
tests/test_help.py:9: in <module>
from tests.conftest import AssertRichFormat, AssertStr, InvokeCli
E   ModuleNotFoundError: No module named 'tests.conftest'
```